tests/test_cli_compile.py

```diff
@@ -510,23 +477,22 @@ def test_dry_run_quiet_option(runner):
     assert not out.stderr_bytes
 
 
-@pytest.mark.network
-def test_generate_hashes_with_editable(runner):
+def test_generate_hashes_with_editable(pip_conf, runner):
     small_fake_package_dir = os.path.join(PACKAGES_PATH, "small_fake_with_deps")
     small_fake_package_url = path_to_url(small_fake_package_dir)
     with open("requirements.in", "w") as fp:
         fp.write("-e {}\n".format(small_fake_package_url))
-        fp.write("pytz==2017.2\n")
     out = runner.invoke(
         cli, ["--generate-hashes", "--index-url", PyPIRepository.DEFAULT_INDEX_URL]
```

Comment:
```suggestion
         cli, ["--generate-hashes"]
```
`--index-url=PyPIRepository.DEFAULT_INDEX_URL` is redundant, since we use `pip_conf` fixture with `no-index=true`.